cf-agent/verify_files.c

```diff
@@ -694,7 +694,7 @@ PromiseResult ScheduleEditOperation(EvalContext *ctx, char *filename, Attributes
         {
             strcpy(edit_bundle_name, (char *) vp);
             args = NULL;
-        }             
```

Comment:
No need to be verbose in the commit message of such a simple commit. Simply write "Fix trailing whitespace" and that's good enough.